.cirrus.yml

```diff
@@ -133,6 +133,19 @@ integration_task:
   setup_script: *setup
   main_script: *main
 
+podman_e2e_task:
+  alias: "podman_e2e"
+    #depends_on:
+    #    - "unit"
```

Comment:
In order for the cache to restore properly, this task should at least depend on `build` at a minimum